config/locales/es.yml

```diff
@@ -170,3 +170,4 @@ es:
   editor: "Editor"
   console: "Consola"
   type_expressions_here: "Acá podés escribir expresiones para probar tu solución"
+  repeat_pending: "Terminar pendientes"
```

Comment:
I think it's more like _next_pending_exercise_, so I would say "Siguiente ejercicio pendiente"